core/src/raw/http_util/multipart.rs

```diff
@@ -231,6 +405,213 @@ content-disposition: form-data; name="submit"
 
 Upload to Amazon S3
 --9431149156168--
+"#;
+
+        assert_eq!(
+            expected,
+            // Rust can't represent `\r` in a string literal, so we
+            // replace `\r\n` with `\n` for comparison
```

Comment:
I'd like to say "rust will replace CRLF in string literals with LF".

Reply:
Thanks, but I prefer to write directly.